lib/dataTrigger/dpnsTriggers/domainCreateDataTrigger.js

```diff
@@ -0,0 +1,63 @@
+const DataTriggerExecutionResult = require('../DataTriggerExecutionResult');
+const DataTriggerExecutionError = require('../../errors/DataTriggerExecutionError');
+const hash = require('../../util/hash');
+const Document = require('../../document/Document');
+const DataTrigger = require('../DataTrigger');
+
+/**
+ * @typedef domainCreateDataTrigger
+ * @param {Document} document
+ * @param {DataTriggerExecutionContext} context
+ * @return {Promise<DataTriggerExecutionResult>}
+ */
+async function domainCreateDataTrigger(document, context) {
+  const dataTriggerExecutionResult = new DataTriggerExecutionResult();
+
+  if (document.getAction() !== Document.ACTIONS.CREATE) {
+    dataTriggerExecutionResult.addError(
+      new DataTriggerExecutionError(document, context, 'Document action is not create'),
```

Comment:
I think it will be simpler for users to throw any type of errors and wrap them outside to `DataTriggerExecutionError`.

Reply:
Already fixed